src/components/ContentListTable/ContentListTable.tsx

```diff
@@ -18,6 +18,7 @@ import {
   Td,
   Th,
   Thead,
+  ThProps,
```

Comment:
Good use of this. It's stuff like this that make me feel Golang pairs well with typescript.

Reply:
>> ThProps['sort']